roles/confluent.variables/defaults/main.yml

```
@@ -58,6 +58,18 @@ common_role_completed: false
 
 proxy_env: {}
 
+### The method of installation. Valid values are "package" or "archive". If "archive" is selected then services will not be installed via the use of yum or apt, but will instead be installed via expanding the target .tar.gz file from the Confluent archive into the path defined by `archive_destination_path`. Configuration files are also kept in this directory structure instead of `/etc`. SystemD service units are copied from the ardhive for each target service and overrides are created pointing at the new paths. The "package" installation method is the default behavior that utilizes yum/apt.
+installation_method: "package"
+
+### The Scala version of the Confluent Platform archive to download. Possible values: 2.11, 2.12, etc. If you don't have a specific version requirement then use the default.
+confluent_archive_scala_version: 2.12
+
+### The path the downloaded archive is expanded into. Using the default with a `confluent_package_version` of *5.5.1* results in the following installation path `/opt/confluent/confluent-5.5.1/` that contains directories such as `bin` and `share`, but may be overridden usinf the `binary_base_path` property.
+archive_destination_path: "/opt/confluent"
+
+### If yes then when performing an archive deployment the SystemD service files will be overwritten with the new paths and options
+overwrite_systemd_services: yes
```

```suggestion
overwrite_systemd_services: true
```

we have a yaml linter that does not like yes/no :)

---

Fixed in d10c97e8